Python/Doc_Ppt_to_Pdf_convertor/doctopdf.py

```diff
@@ -0,0 +1,14 @@
+import sys
```

Comment:
Code plagiarized from [here](https://stackoverflow.com/questions/6011115/doc-to-pdf-using-python).

I understand the script is minimalistic, but a word-to-word copy? No can't merge this.

Reply:
yes, it's plagiarized but I have been using these from last 3 years. I saw this project and thought of sharing here. Thanks

Reply:
I shared here because these scripts has helped me a lot and these are a lot faster then those online tools.